docker/linux/base/build/Dockerfile

```diff
@@ -123,6 +123,10 @@ RUN cd /tmp \
     && echo 16-init-17653-g39da55e8-2 >> ${CLANG_16_TC_HOME}/cr_build_revision \
     && rm clang-llvmorg-16-init-17653-g39da55e8-2.tgz
 
+# === Install build time python dependencies
+COPY base/build/requirements.txt /opt/requirements.txt
+RUN python3 -m pip install --require-hashes --no-deps -r /opt/requirements.txt
```

Comment:
I think you are doing a no-op refactor, but given that this moved here.. shouldn't it just become an apt `python3-certifi` package above in this file ? ( note `python-certifi` probably isn't needed, Python2 should be gone )

Reply:
Perhaps not, i checked the `debian11` and it has `python3-certifi/oldstable 2020.6.20-1`

Reply:
Looks like it worked.